### DISCUSSION TITLE
"too much data...chunk size xxx exceeds chunkSizeLimit" error

### DISCUSSION THREAD

**Discussion by hkyy1020** [GMOD/jbrowse-components#2000] [Show and tell]
When I view my .maf file, I get the error "Too much data...chunk size xxx exceeds chunkSizeLimit".  But when I edited my trackList.json, changing the chunkSizeLimit to 10000000, the  chunkSizeLimit is still showing 20000 in the website. Please kindly tell me what I can do to solve the problems. Thanks so much.

**Comment by cmdcolin** [comment]
just for reference this is the forum for jbrowse 2, but this appears to be a jbrowse 1 issue

can you post your config? you may also try clearing cache to see if it is updated in the web then

**Reply by zhangya01** [reply]
I am also trying to display my .maf file, and I get an error "Too much data...chunk size xxxxx exceeds chunkSizeLimit". And I want to know how can I increase the chunkSize. Thank you.

**Reply by cmdcolin** [reply]
can you clarify what you mean by MAF? are you using https://github.com/cmdcolin/mafviewer ?

**Reply by cmdcolin** [reply]
if you are using mafviewer can you post to the issues forum there?

**Reply by zhangya01** [reply]
Yes, I am using mafviewer.
